pipeline/extract.py

```diff
@@ -0,0 +1,198 @@
+'''File used for extracting information from bandcamp's website and API'''
+
+from time import sleep
+import json
+import logging
+import asyncio
+import aiohttp
+import requests as req
+from bs4 import BeautifulSoup
```

Comment:
The ordering is good here, but having them ordered alphabetically within their groups (Standard, Third-party) would improve readability.